docs/jwst/stpipe/devel_io_design.rst

```diff
@@ -96,18 +96,17 @@ the result will be saved in a file called::
 Similarly, the same code can be used in a Python script or interactive
 environment as follows::
 
-.. doctest_skip::
-
-  >>> import jwst
-  >>> input = jwst.datamodels.open('input_data.fits')
-  >>> result = MyStep.call(input)
-      # `result` contains the resulting data
-      # which can then be used by further `Steps`'s or
-      # other functions.
-      #
-      # when done, the data can be saved with the `DataModel.save`
-      # method
-  >>> result.save('my_final_results.fits')
+  >>>
+  >> import jwst
+  >> input = jwst.datamodels.open('input_data.fits')
+  >> result = MyStep.call(input)
+     # `result` contains the resulting data
+     # which can then be used by further `Steps`'s or
+     # other functions.
+     #
+     # when done, the data can be saved with the `DataModel.save`
+     # method
+  >> result.save('my_final_results.fits')
```

Comment:
Shouldn't this be `>>>` and `...` for continuation (if necessary)? However, I believe each comment line is a `>>> `

Reply:
I've explicitly set it this way so that the doctest doesn't run on the block.  This seems to be the only way to fool it to do so without having `# doctest: +SKIP` on each line and thus cluttering up the example.

Reply:
Ah, there is a better way to do this.  The `# doctest: +SKIP` does work, and does not show up in the HTML output.  And I only have to add it to 3 lines.  Fixing.  Thanks @mcara.